prml/consortium-permission/src/lib.rs

```diff
@@ -302,6 +305,19 @@ impl<T: Trait> Module<T> {
         }
     }
 
+    /// Counts the number of accounts that have been granted a specific permission.
+    /// Takes a topic and value, iterate through all existing claims, and count the
+    /// numbers of claims with matching topic and value.
+    pub fn granted_permission_count(topic: &Topic, value: &Value) -> u32 {
+        let mut count = 0;
+        Claim::<T>::iter().map(|((holder, t), (issuer, v))|{
+            if &t == topic && &v == value {
+                count+=1;
+            }
+        });
+        count
```

Comment:
could use functional style like 💭 
```suggestion
Claim::<T>::iter().filter(|((_holder, t), (_issuer, v))| &t == topic && &v == value).count()
```